src/components/DatePicker/DatePickerOptions.tsx

```diff
@@ -124,6 +143,7 @@ export const DatePickerOptions = ({
       optionsComponents.push(getOptionComponent(option))
     })
   }
+  optionsComponents.push(...customOptionComponents)
```

Comment:
Could we make the custom option components the only ones shown here (remove `this-month`, `last-month`, etc) and just set the default value for custom options to include those such that users can disable `this-month`, or `last-month` or any of the other preset options by excluding them?